package.json

```diff
@@ -1,25 +1,20 @@
 {
-  "name": "llama-tokenizer-js",
-  "version": "1.1.3",
-  "description": "JS tokenizer for LLaMA-based LLMs",
-  "main": "llama-tokenizer.js",
+  "name": "llama-tokenizer",
```

Comment:
package name changed by accident?

Reply:
this  one is root package.json for the monorepo and as we can't have two workspaces with same package name thus removed js from the root package.json because the what is going to be published is in the `packages/llama-tokenizer` directory